lib/kiev/rack/request_logger.rb

```diff
@@ -117,16 +117,29 @@ def form_data(request:, began_at:, status:, env:, body:, response:, exception:)
           data[:body] = full_body.join
         end
 
+        if data[:body] && !data[:body].empty? && response_gzipped?(response)
```

Comment:
this code should be wrapped inside the `if config.log_response_body_condition.call(request, response)` condition above.